docs/api/covid_hosp.md

```diff
@@ -149,25 +157,31 @@ The following sample shows how to import the library and fetch MA on 2020-05-10
 ### Python
 
 Optionally install the package using pip(env):
-````bash
+```bash
 pip install delphi-epidata
-````
+```
 
 Otherwise, place `delphi_epidata.py` from this repo next to your python script.
 
-````python
+```python
 # Import
 from delphi_epidata import Epidata
 # Fetch data
 res = Epidata.covid_hosp('MA', 20200510)
 print(res['result'], res['message'], len(res['epidata']))
-````
+```
 
 # Repair Log
 
 If we ever need to repair the data record due to a bug in our code (not at the
 source), we will update the list below.
 
+## October 21, 2021
+
+All issues between 20210430 and 20211021 were re-uploaded to include new columns added by
```

Comment:
-> 20221021?

Reply:
alas, this was actually last year's patch that we apparently never documented